Cargo.toml

```diff
@@ -23,20 +23,24 @@ keywords = ["io", "async", "non-blocking", "futures"]
 
 members = [
   "./",
+  "tokio-executor",
   "tokio-io",
+  "tokio-threadpool",
 ]
 
 [badges]
 travis-ci = { repository = "tokio-rs/tokio" }
 appveyor = { repository = "carllerche/tokio" }
 
 [dependencies]
+tokio-io = "0.1"
```

Comment:
Why do you set path in `tokio-executor` like `path = "tokio-executor"` but you get `tokio-io` from crates.io?

Reply:
The tokio-io dependency probably should be updated as well 👍 That can be done on master.